test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -209,17 +217,19 @@ class StructuredFormDataBuilderSpec extends Spec {
         "3_field-year"  -> "6"
       ),
       objectStructure(
-        "field" -> arrayNode(
-          objectStructure(
-            "day"   -> textNode("1"),
-            "month" -> textNode("2"),
-            "year"  -> textNode("3")
+        field(
+          "field",
+          arrayNode(
+            objectStructure(
+              field("day", textNode("1"), Map.empty),
+              field("month", textNode("2"), Map.empty),
+              field("year", textNode("3"), Map.empty)),
+            objectStructure(
+              field("day", textNode("4"), Map.empty),
+              field("month", textNode("5"), Map.empty),
+              field("year", textNode("6"), Map.empty))
           ),
-          objectStructure(
-            "day"   -> textNode("4"),
-            "month" -> textNode("5"),
-            "year"  -> textNode("6")
-          )
+          Map.empty
```

Comment:
Make our lives easy & remove the duplication by giving the alternateFieldNames parameter of field(...) a default, Map.empty.